de/software/openshift.md

```diff
@@ -8,7 +8,20 @@ tags:
 - support
 - infrastruktur
 ---
-Auf der Kubernetes Plattform __Openshift__ werden ca. 100 interne, als auch externe Services betrieben.
+Auf der Kubernetes Plattform __Openshift__ werden interne, als auch externe Services betrieben.
```

Comment:
shouldn't we mention "Red Hat"? like 

Auf der Kubernetes Plattform "Red Hat OpenShift" ....

Reply:
i would not want to name any brands